### ISSUE TITLE
Automatic Workflow Deploy

### ISSUE THREAD

**Issue by 0x4007** [ubiquity-os/plugins-wishlist#55]
If no changes were made to any of the .github/workflows/*.yml files then we should automatically run the workflows. 

This is relevant for QA (build tests, continuous deploys etc.)

**Comment by 0x4007** [comment]
@gentlementlegen how long does it take to make plugins? I assume the core logic of this is only a few lines.

**Comment by gentlementlegen** [comment]
With the SDK it's two lines! However I am not sure to understand your spec.

**Comment by gentlementlegen** [comment]
The reason behind it is because it is potentially running malicious code. I think this behavior can be changed in GitHub settings.

**Comment by gentlementlegen** [comment]
![image](https://github.com/user-attachments/assets/4cb1a976-9c1b-4bbb-bdff-bb9d5a4370eb)
https://github.com/ubiquity-os/plugins-wishlist/settings/actions

**Comment by 0x4007** [comment]
Yes I am aware it can be manually changed for every repo but isn't that annoying to keep track of given that we make a new repo every 1-2 weeks?

**Comment by gentlementlegen** [comment]
You can change this org wise so all subsequent repos will have that same rule defined: https://github.com/organizations/ubiquity-os/settings/actions. Sadly doesn't apply for previously created repos, but a script could be done about that. I think that should solve the problem?

**Comment by 0x4007** [comment]
This is already as permissive as can be 

![image](https://github.com/user-attachments/assets/df3c6905-f730-4671-9879-a52d3a5e6a39)

**Comment by 0x4007** [comment]
It might be enough I'm not sure yet
